uaclient/status.py

```diff
@@ -621,25 +624,30 @@ def format_tabular(status: Dict[str, Any]) -> str:
                 content.append(STATUS_SIMULATED_TMPL.format(**service))
             return "\n".join(content)
 
-        content = [
-            STATUS_UNATTACHED_TMPL.format(
-                name="SERVICE",
-                available="AVAILABLE",
-                description="DESCRIPTION",
-            )
-        ]
-        for service in status["services"]:
-            descr_override = service.get("description_override")
-            description = (
-                descr_override if descr_override else service["description"]
-            )
-            content.append(
+        if not status["services"]:
+            content = [messages.STATUS_NO_SERVICES_AVAILABLE]
```

Comment:
I think can just return the `content` here directly

Reply:
Well, there may be notices, for instance, or features to report - so the idea is that this piece of code replaces only the services table.
Is there a specific reason not to show anything else than the message?

Reply:
Yeah, my rationale was that if we don't have any services available, there is no other information we can provide. But you are right, maybe some notices will be useful (even though it would be odd for them to appear, but better be safe than sorry)